libs/domain/cart/services/src/default-cart.service.ts

```diff
@@ -287,6 +296,10 @@ export class DefaultCartService implements CartService {
     return this.executeWithOptionalCart(qualifier, this.addCouponCommand$);
   }
 
+  deleteCoupon(qualifier: CouponQualifier): Observable<unknown> {
```

Comment:
and need to add this method to the cart mock

Reply:
Coupon service will be set in different service and adapter in a follow up story, pls check: https://spryker.slack.com/archives/C029XPSBXC6/p1704820725966759